### DISCUSSION TITLE
Trusted-Set-Attr is not setting valued in some cases.

### DISCUSSION THREAD

**Discussion by ghost** [uBlockO/uBO-Scriptlets#28] [General] [answered]
I found some cases where the attribute is not set, for example using `*` as selector will only set it in few elements on top of the tree of nodes but then it just stops, and don't add it to all.

For example using `search.brave.com##+js(tsa, target, _blank, *)` it is only added to `<body class="is-pc">`, and then `<style type="text/css">` and `<div style="display: contents">` but nothing further inside of them.

So using `search.brave.com##+js(tsa, target, _blank, a)` will not be added to any search result.

if you want to compare it to the set-attr in uBlock. if you do a `search.brave.com##+js(set-attr, target, 1, *)` the attr will be set as expected, but of course, it is useless if it has limited values.

Anyway, just letting you know, if you want to check that.

Thanks for your scriptlets, they are useful. Have a good day.

**Comment by uBlock-user** [comment, marked as answer]
@ghost Use `search.brave.com##+js(tsa, target, _blank, *, loop)`

> `search.brave.com##+js(set-attr, target, 1, *)`

That doesn't work at all on Firefox for me.
